apps/challenges/aws_utils.py

```diff
@@ -430,8 +441,15 @@ def register_task_def_by_challenge_pk(client, queue_name, challenge):
 
         challenge_aws_keys = get_aws_credentials_for_challenge(challenge.pk)
         if challenge.is_docker_based:
-            from .models import ChallengeEvaluationCluster
+            from .models import ChallengeEvaluationCluster, ChallengePhase
 
+            challenge_phases = ChallengePhase.objects.filter(
+                challenge=challenge.pk
+            )
+            efs_id = ""
+            if challenge_phases.count() > 0:
+                efs_id = challenge_phases[0].efs_id
```

Comment:
Why are we taking only the `0`th index?

Reply:
For now, we want to just set up 1 EFS per challenge right?